packages/open-next/src/build/edge/createEdgeBundle.ts

```diff
@@ -23,7 +26,11 @@ interface BuildEdgeBundleOptions {
   entrypoint: string;
   outfile: string;
   options: BuildOptions;
-  overrides?: OverrideOptions;
+  overrides?: OverrideOptions & {
+    originResolver?:
```

Comment:
Is it used there?

Reply:
Yeah, if you define a custom `originResolver` in the config for the external middleware, it will replace the default one with the dummy in the final bundle (since it's not used)

Reply:
But l68+ should be update too, no?

Reply:
Ho right, i missed it